src/PVRIptvData.cpp

```diff
@@ -77,13 +77,36 @@ inline bool GetAttributeValue(const xml_node<Ch> * pNode, const char* strAttribu
   return true;
 }
 
+// http://stackoverflow.com/a/17708801
+static const std::string UrlEncode(const std::string &value) {
```

Comment:
please change your code style according to the guidelines https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#31-braces